docs/releases/5.0.txt

```diff
@@ -299,9 +299,9 @@ File Uploads
 Forms
 ~~~~~
 
-* :attr:`.ChoiceField.choices` now accepts
-  :ref:`Choices classes <field-choices-enum-types>` directly instead of
-  requiring expansion with the ``choices`` attribute.
+* :attr:`.Field.choices` now accepts mappings as well as iterables, and
+  also accepts :ref:`Choices classes <field-choices-enum-types>` directly
+  instead of requiring expansion with the ``choices`` attribute.
```

Comment:
I'm thinking this could go be moved up to the "what's new" section? 🤔

Reply:
Good idea! You refer to the whole paragraph, right?

Reply:
Note that this is in there twice - once under forms and once under models. I wonder if the links point to the correct `Field` in each case... 🤔 Let me know what you want to do - whether we want to hoist this up as a headline feature or not.

Reply:
I'll cite https://github.com/django/django/pull/11540/files which was a similar change to this and also a headline feature. I was a bit unsure about suggesting this (my first time suggesting promotion to headline), but being able to cite this PR gives some precident. 

I was going to say it would be nice as a headline feature with an example saying you used to do x... now you can do y.... 

... but 11540 was words, without example. So likely that's fine too!

Reply:
I'm working on the final touches and I'm proposing something like this for the What's New promotion:
![image](https://github.com/django/django/assets/124304/cd2d797e-9351-4f05-92fb-f327dd032609)